src/panels/lovelace/ha-panel-lovelace.ts

```diff
@@ -295,7 +301,7 @@ class LovelacePanel extends LitElement {
       },
       saveConfig: async (newConfig: LovelaceConfig): Promise<void> => {
         const { config: previousConfig, mode: previousMode } = this.lovelace!;
-        this._checkLovelaceConfig(newConfig);
+        newConfig = this._checkLovelaceConfig(newConfig);
```

Comment:
We should freeze the config again here, as it is updated with copies and will not be frozen completely anymore.

Reply:
![image](https://user-images.githubusercontent.com/5662298/74463377-0c7e0a80-4e92-11ea-9512-c2199ae72a56.png)

Reply:
`_checkLovelaceConfig` should return a frozen object.